token/request.go

```diff
@@ -276,7 +278,7 @@ func (r *Request) Issue(wallet *IssuerWallet, receiver Identity, typ string, q u
 // The action transfers tokens of the passed types to the receivers for the passed quantities.
 // In other words, owners[0] will receives values[0], and so on.
 // Additional options can be passed to customize the action.
-func (r *Request) Transfer(wallet *OwnerWallet, typ string, values []uint64, owners []Identity, opts ...TransferOption) (*TransferAction, error) {
+func (r *Request) Transfer(context context.Context, wallet *OwnerWallet, typ string, values []uint64, owners []Identity, opts ...TransferOption) (*TransferAction, error) {
```

Comment:
for consistency: you're using `ctx` in other places (and I like it a bit more, because the name `context` shadows the name of the package.